angular/projects/spark-angular/src/public-api.ts

```diff
@@ -74,8 +74,8 @@ export { SprkFormatterPhoneNumberDirective } from './lib/directives/inputs/forma
 export { SprkFormatterPhoneNumberModule } from './lib/directives/inputs/formatters/sprk-formatter-phone-number/sprk-formatter-phone-number.module';
 export { SprkFormatterSsnDirective } from './lib/directives/inputs/formatters/sprk-formatter-ssn/sprk-formatter-ssn.directive';
 export { SprkFormatterSsnModule } from './lib/directives/inputs/formatters/sprk-formatter-ssn/sprk-formatter-ssn.module';
-export { SprkButtonDirective } from './lib/directives/inputs/sprk-button/sprk-button.directive';
-export { SprkButtonModule } from './lib/directives/inputs/sprk-button/sprk-button.module';
+export { SprkButtonDirective } from './lib/directives/sprk-button/sprk-button.directive';
```

Comment:
Changing the path could be a breaking change if teams are importing it from this spot. In the past we had a workaround for this based on the way we exported it. I forget it exactly, you could dig around this file to find out how we did it.

Reply:
I don't see where we're doing that in this file. I would expect the app cares more about the import name than the directory it's hooked onto.

To clarify, do you mean if someone is importing `../lib/directives/inputs/sprk-button/sprk-button.module` directly?
If that's the case, I could see it would break there...but apps should be importing `SprkButtonModule`.
Importing it directly like that shouldn't be supported

Reply:
We'll mention this in the release notes!